README.md

```diff
@@ -114,6 +114,34 @@ dependencies {
 }
 ```
 
+## Development
+### Setup
+* development requires java 11 to be installed
+* development assumes you are running on Ubuntu, while you may be able to use other operating systems they are not supported for developement.
```

Comment:
Do we need this sentence? I'm developing on macOS, and it seems working as expected.
If we know any trouble with Windows or other OS, I want to fix it.

Reply:
Ok. I based it on the fact that I ran into errors when doing the functional testing in my Windows machine and that we use the Ubuntu image on the CI server. When I get a chance I will update it.